src/WebGL/Browser.WebGL.fs

```diff
@@ -0,0 +1,909 @@
+    abstract TEXTURE25: float with get, set
+    abstract TEXTURE26: float with get, set
+    abstract TEXTURE27: float with get, set
+    abstract TEXTURE28: float with get, set
+    abstract TEXTURE29: float with get, set
+    abstract TEXTURE3: float with get, set
+    abstract TEXTURE30: float with get, set
+    abstract TEXTURE31: float with get, set
+    abstract TEXTURE4: float with get, set
+    abstract TEXTURE5: float with get, set
+    abstract TEXTURE6: float with get, set
+    abstract TEXTURE7: float with get, set
+    abstract TEXTURE8: float with get, set
+    abstract TEXTURE9: float with get, set
+    abstract TEXTURE_2D: float with get, set
+    abstract TEXTURE_BINDING_2D: float with get, set
+    abstract TEXTURE_BINDING_CUBE_MAP: float with get, set
+    abstract TEXTURE_CUBE_MAP: float with get, set
+    abstract TEXTURE_CUBE_MAP_NEGATIVE_X: float with get, set
+    abstract TEXTURE_CUBE_MAP_NEGATIVE_Y: float with get, set
+    abstract TEXTURE_CUBE_MAP_NEGATIVE_Z: float with get, set
+    abstract TEXTURE_CUBE_MAP_POSITIVE_X: float with get, set
+    abstract TEXTURE_CUBE_MAP_POSITIVE_Y: float with get, set
+    abstract TEXTURE_CUBE_MAP_POSITIVE_Z: float with get, set
+    abstract TEXTURE_MAG_FILTER: float with get, set
+    abstract TEXTURE_MIN_FILTER: float with get, set
+    abstract TEXTURE_WRAP_S: float with get, set
+    abstract TEXTURE_WRAP_T: float with get, set
+    abstract TRIANGLES: float with get, set
+    abstract TRIANGLE_FAN: float with get, set
+    abstract TRIANGLE_STRIP: float with get, set
+    abstract UNPACK_ALIGNMENT: float with get, set
+    abstract UNPACK_COLORSPACE_CONVERSION_WEBGL: float with get, set
+    abstract UNPACK_FLIP_Y_WEBGL: float with get, set
+    abstract UNPACK_PREMULTIPLY_ALPHA_WEBGL: float with get, set
+    abstract UNSIGNED_BYTE: float with get, set
+    abstract UNSIGNED_INT: float with get, set
+    abstract UNSIGNED_SHORT: float with get, set
+    abstract UNSIGNED_SHORT_4_4_4_4: float with get, set
+    abstract UNSIGNED_SHORT_5_5_5_1: float with get, set
+    abstract UNSIGNED_SHORT_5_6_5: float with get, set
+    abstract VALIDATE_STATUS: float with get, set
+    abstract VENDOR: float with get, set
+    abstract VERSION: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_BUFFER_BINDING: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_ENABLED: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_NORMALIZED: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_POINTER: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_SIZE: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_STRIDE: float with get, set
+    abstract VERTEX_ATTRIB_ARRAY_TYPE: float with get, set
+    abstract VERTEX_SHADER: float with get, set
+    abstract VIEWPORT: float with get, set
+    abstract ZERO: float with get, set
+    [<Emit("new $0($1...)")>] abstract Create: unit -> WebGLRenderingContext
+
+namespace Browser
+
+open Browser.Types
+open Fable.Core
+
+module WebGL =
```

Comment:
We usually add the `AutoOpen` attribute to the API of the Browser modules, so users can automatically access their contents with just "open Browser".